lightly/loss/ntx_ent_loss.py

```diff
@@ -117,8 +94,11 @@ def forward(self,
             Args:
                 out0:
                     Output projections of the first set of transformed images.
+                    Shape: (batch_size, embedding_size)
                 out1:
                     Output projections of the second set of transformed images.
+                    Shape: (batch_size, embedding_size)
+                    out0[i] and out1[i] are two different augmentation of the same image.
```

Comment:
Would drop this line. Whether we have two views of the same image or two different views is up to the user.